### PR TITLE
change menu page

### PR DESCRIPTION
**Pull request by AZXEthan**

### CODE REVIEW

**Review by zsun122 — APPROVED** [review]

File: menu-gen-frontend/src/pages/MenusPage.tsx [hunk 1]
```diff
@@ -58,55 +60,55 @@ const MenusPage = () => {
...
+                    height={256}
+                />
+              </CardHeader>
+              <CardBody className="flex flex-col gap-3 items-center">
+                {menu.food_items.map((item) => (
+                    <Card key={item.title} className="mb-4 flex flex-col items-center w-full">
+                      <h3 className="text-xl font-bold">{item.title}</h3>
+                      <p>{item.description}</p>
+                      <p>{'$' + (item.price ? item.price : '0')}</p>
+                      {/* ... other item properties ... */}
```
Comment (jamesli12) [inline menu-gen-frontend/src/pages/MenusPage.tsx:83]: Why is this comment still here?

File: menu-gen-frontend/src/pages/MenusPage.tsx [hunk 2]
```diff
@@ -58,55 +60,55 @@ const MenusPage = () => {
...
+                ))}
+              </CardBody>
+              <CardFooter className="flex flex-col items-center">
+                <Link href={`/menu/${menu._id}`}>
+                  <a className="inline-block mt-4 px-4 py-2 bg-blue-500 text-white rounded">
+                    View Menu
+                  </a>
+                </Link>
+                <div className="mt-4">
+                  {/* QR Code generation */}
```
Comment (jamesli12) [inline menu-gen-frontend/src/pages/MenusPage.tsx:100]: Same thing here, why is this here?

File: menu-gen-frontend/src/pages/MenuPage.tsx [hunk 3]
```diff
@@ -38,41 +47,48 @@ const MenuPage = () => {
...
-                alt={item.title}
-                width={128}
-                height={128}
-              />
-            </Card>
-          ))}
-        </CardBody>
-      </Card>
-    </div>
+      <div className="min-h-screen p-5" style={{ backgroundColor: 'black' }}> {/* Background set to yellow */}
```
Comment (jamesli12) [inline menu-gen-frontend/src/pages/MenuPage.tsx:50]: This entire file just has the auto-generated comments lol

**Review by jamesli12 — APPROVED** [review]
lgtm
